app/scripts/controllers/app-state-controller.ts

```diff
@@ -179,89 +184,239 @@ const getDefaultAppStateControllerState = (
   newPrivacyPolicyToastClickedOrClosed: null,
   newPrivacyPolicyToastShownDate: null,
   hadAdvancedGasFeesSetPriorToMigration92_3: false,
-  ...initState,
-  qrHardware: {},
-  nftsDropdownState: {},
   surveyLinkLastClickedOrClosed: null,
-  signatureSecurityAlertResponses: {},
-  switchedNetworkDetails: null,
   switchedNetworkNeverShowMessage: false,
-  currentExtensionPopupId: 0,
+  ...getDefaultAppStateOverrides(),
 });
 
-export class AppStateController extends EventEmitter {
-  private readonly extension: AppStateControllerOptions['extension'];
+function getDefaultAppStateOverrides() {
+  return {
+    qrHardware: {},
+    nftsDropdownState: {},
+    signatureSecurityAlertResponses: {},
+    switchedNetworkDetails: null,
+    currentExtensionPopupId: 0,
```

Comment:
This was the original set of properties:

``` typescript
qrHardware: {},
  nftsDropdownState: {},
  surveyLinkLastClickedOrClosed: null,
  signatureSecurityAlertResponses: {},
  switchedNetworkDetails: null,
  switchedNetworkNeverShowMessage: false,
  currentExtensionPopupId: 0,
```

So it seems we are missing `surveyLinkLastClickedOrClosed`:

``` suggestion
    qrHardware: {},
    nftsDropdownState: {},
    surveyLinkLastClickedOrClosed: null,
    signatureSecurityAlertResponses: {},
    switchedNetworkDetails: null,
    currentExtensionPopupId: 0,
```

Reply:
yes two properties where removed as per https://github.com/MetaMask/metamask-extension/pull/28784#discussion_r1862809521